llvm/lib/Target/AArch64/AArch64Subtarget.h

```diff
@@ -58,6 +58,7 @@ class AArch64Subtarget final : public AArch64GenSubtargetInfo {
     CortexA55,
     CortexA510,
     CortexA520,
+    CortexA520AE,
```

Comment:
These might not be worth adding, considering they should be the same as CortexA520, and could reuse the same enum.

Reply:
Yes, good idea. Amended.